test/internet/test-dns-ipv6.js

```diff
@@ -44,46 +40,49 @@ function checkWrap(req) {
 }
 
 TEST(function test_resolve6(done) {
-  var req = dns.resolve6('ipv6.google.com', function(err, ips) {
-    if (err) throw err;
+  const req = dns.resolve6('ipv6.google.com',
+    common.mustCall((err, ips) => {
```

Comment:
well... that way has problems with the linter as it is expecting the code to be like this:

```javascript
const req = dns.resolve6('ipv6.google.com',
                            common.mustCall((err, ips) => {
                              assert.ifError(err);

                              assert.ok(ips.length > 0);

                              for (let i = 0; i < ips.length; i++)
                                assert.ok(isIPv6(ips[i]));

                              done();
                            }));```

in the previous way I didn't have linter problems... just let me know what to do here then to keep the current or ident the whole function code... 

I implemented all the other suggestions 🙂